knowledge-base/actions/crazy-max/ghaction-github-pages/action-security.yml

```diff
@@ -0,0 +1,2 @@
+name: 'GitHub Pages'
+#No permissions Needed
```

Comment:
I think there is `content: write` permission `to deploy code to github pages`, view [here](https://github.com/crazy-max/ghaction-github-pages/blob/0c20c87f978dc85e4b78d5bf7c156de15b2f169e/src/main.ts#L134)